### PR TITLE
fix: default value for webhook_content_type

### LINKED ISSUE #4: application/json is invalid default value for webhook_content_type
### what

- `application/json` is an incorrect default value to use for the `webhook_content_type` variable

### why

- The only supported values for `content_type` in the github api are `json` & `form`
  https://developer.github.com/v3/repos/hooks/#parameters  
  image

### PR DESCRIPTION
**Pull request by joppa27**
fixes #4

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -30,8 +30,8 @@ variable "webhook_url" {
 }
 
 variable "webhook_content_type" {
-  description = "Webhook Content Type (E.g. application/json)"
-  default     = "application/json"
+  description = "Webhook Content Type (E.g. json)"
+  default     = "json"
```
Comment (osterman) [inline variables.tf:34]: Are you sure this is right? This doesn’t look like a valid content type.

File: README.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ Available targets:
 | github_repositories | List of repository names which should be associated with the webhook | list | `<list>` | no |
 | github_token | GitHub token used for API access | string | - | yes |
 | name | The type of webhook | string | `web` | no |
-| webhook_content_type | Webhook Content Type (E.g. application/json) | string | `application/json` | no |
+| webhook_content_type | Webhook Content Type (E.g. json) | string | `json` | no |
```
Comment (osterman) [inline README.md:88]: We need to rebuild the readme using make

make init 
make readme/deps
make readme

**Review by osterman — CHANGES_REQUESTED** [review]
Thanks for resubmitting. Didn’t want to subject you to the rebuild process. If you are up to it, please see inline comments. Readme is computed dynamically and automatically pulls in descriptions from values.

File: variables.tf (hunk 1 above)
Comment (joppa27) [reply variables.tf:34]: @osterman yes, I am sure this is right. See #4 `application/json`, is an "invalid type" in this case.

File: README.md (hunk 2 above)
Comment (joppa27) [reply README.md:88]: @osterman hrmmm I ran ☝️, but something is wrong with the makefile? It overwrote some changes I made to README.md, had to manually fix that too 😅

Comment (eosterman-rms) [reply README.md:88]: Sorry for the confusion. So our `README.md` is rendered from the `README.yaml`. All terraform input and output descriptions are generated by `terraform-docs`. So if the `README.md` is not correct, then check these other places.

**Review by osterman — APPROVED** [review]

**Comment by osterman** [comment]
thanks @joppa27 !

**Pull request merged**